3/2-Datamodel.md

```diff
@@ -0,0 +1,144 @@
+
+## Section 1: Datamodel
+<a name="top"/>
+
+Central to the OpenCannabis project's goals is a *unified datamodel* for use across functional boundaries. When
+cultivators talk about *flowers* with labs, and labs talk about *test results* with retailers, and so on, they are all
+talking about the same thing - but in today's legal cannabis industry, they aren't speaking the same language.
+
+### Table of Contents
+1. [Toolchain](#Toolchain)
+1. [General Guidelines](#General-Guidelines)
+    - [Nomenclature](#Nomenclature)
+1. [Message Categories](#Message-Categories)
+    - [Part 1.1: Core Structures](3-Core-Structures.md)
+        - `base`: Foundational structures used across the spec.
+        - `temporal`: Dates, times, and so on.
+        - `crypto`: Cryptographic primitives and containers.
```

Comment:
the datamodel is split into two parts:
- **`Part 1.1` (Section 1, Part 1): Core Structures**
    These models are used across the spec, and are generally considered pivotal to the concept of cannabis. They are defined explicitly in the specification.
- **`Part 1.2` (Section 1, Part 2): Extension Structures**
    These models are defined by _Specification Extensions_, which are domain-specific, or functionality-specific, models and structures with _Working Groups_ overseeing their ratification.